corehq/apps/style/static/style/js/hq.helpers.js

```diff
@@ -1,39 +1,46 @@
-$(function() {
-    var clearAnnouncement = function (announcementID) {
-        $.ajax({
-            url: '/announcements/clear/' + announcementID
-        });
-    };
+/* global define, jQuery */
+(function (factory) {
+    if (typeof define === 'function' && define.amd) {
+        define(['jquery', 'knockout'], factory);
+    } else {
+        factory(jQuery, ko);
+    }
+}(function ($, ko) {
```

Comment:
Getting used to seeing this form of module preamble now. It took me a bit to figure out what was going on. Wow, that is ... trippy. Wonder if we could write our own `define` adapter for `hqDefine` so we could use AMD syntax everywhere that wants to support both RequireJS and hqDefine?